### DISCUSSION TITLE
Set the height of the dropdown list of the Select component?

### DISCUSSION THREAD

**Discussion by wiredmatt** [chakra-ui/chakra-ui#3942] [Q&A]
I'm using the [Select](https://chakra-ui.com/docs/form/select) component, and have a list of 20 items, looking like this:
![image](https://user-images.githubusercontent.com/29718978/116838246-12016680-aba4-11eb-8679-232ecdb52803.png)

As you can see it goes outside of the modal in which it's being wrapped, so I was looking for a way to set the height of that dropdown list, but couldn't find any prop that may do the job, any suggestions?

**Comment by AguChaves92** [comment]
hi im having a similar problem, did you find a workaround?

**Reply by wiredmatt** [reply]
Hey, it's been a while since I last worked on this, please check here: https://github.com/sushi-night/sushi-night-desktop/blob/main/src/react/components/UpdateEntry.tsx

I believe I solved it but honestly I don't remember.
